pages/linux/iostat.md

```diff
@@ -1,19 +1,23 @@
 # iostat
 
-> Report stats for devices and partitions.
+> Report statistics for devices and partitions.
 
-- Display disk statistics with disk IDs in human readable format:
+- Display a report of CPU and disk statistics since system startup:
 
-`iostat -h`
-
-- Display disk statistics with disk names (including LVM) in human readable format:
-
-`iostat -Nh`
+`iostat`
 
 - Display CPU statistics:
 
 `iostat -c`
 
-- Display extended disk statistics with disk names:
+- Display disk statistics with disk names (including LVM):
+
+`iostat -N`
+
+- Display extended disk statistics with disk names for device "sda":
+
+`iostat -xN {{sda}}`
+
+- Display incremental reports of CPU and disk statistics every "interval" second(s) with units converted to megabytes:
 
-`iostat -xN`
+`iostat -m {{interval}}`
```

Comment:
6 examples is ok -- it's more of a guideline than a hard limit. If -m isn't related to the incremental reports, I would say it's preferable to have an extra example, yes.